lib/ret_web/controllers/page_controller.ex

```diff
@@ -229,25 +229,32 @@ defmodule RetWeb.PageController do
   end
 
   def render_index(conn) do
-    {app_config, app_config_script, app_config_csp} = generate_app_config()
+    if conn |> get_req_header("x-original-method") |> Enum.at(0) === "HEAD" do
+      conn
+      |> put_resp_header("hubs-server", "true")
```

Comment:
given that we're calling these servers "hubs", maybe this is redundant and should be "Is-Hub" or something

Reply:
maybe `Hubs-Origin`?

Reply:
what about a header akin to `Content-Type`, but identifies URLs as the root homepage (the 'hub'), avatars, scenes, or rooms?

`Entity-Type`?

Reply:
Yeah, "is-hub" seems okay. I was considering "is-moz-hub" or "is-hub-compatible" to help reduce the chance of future collision with the wider web, but I'm not sure. I guess we probably want to avoid any branding though.